src/AzureDeprecation.Notices.Management/IssueFactory.cs

```diff
@@ -55,17 +55,23 @@ private static void WriteIntro(NewAzureDeprecationV1Message newNoticeV1MessageQu
 
         private static void WriteNotice(NewAzureDeprecationV1Message newNoticeV1MessageQueueMessage, StringBuilder issueBuilder)
         {
-            issueBuilder.AppendLine("### Notice");
-            issueBuilder.AppendLine();
-            issueBuilder.AppendLine(newNoticeV1MessageQueueMessage.Notice.Description);
+            if (!string.IsNullOrWhiteSpace(newNoticeV1MessageQueueMessage.Notice?.Description))

```

Comment:
Can you revert this please?

We should always create this section. If there is no notice; we should add that in the 3rd append line

Reply:
Adjusted, but with TODO for proper wording